### PR TITLE
MASVS 1.2 French translation

### PR DESCRIPTION
**Pull request by A-AFTAHI**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [x] Your PR is linked to an issue.
- [x] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [x] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #321.

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

**Comment by commjoen** [comment]
Great! Can you doublecheck whether the references are done correctly?
@romualdszkudlarek can you review?

File: Document-fr/0x06-V1-Architecture_design_and_threat_modelling_requireme.md [hunk 1]
```diff
@@ -32,9 +32,10 @@ Les exigences pour MASVS-L1 et MASVS-L2 sont listées ci-dessous.
 
 Pour de plus amples informations, il est possible de consulter aussi :
 
-- OWASP Mobile Top 10: M10 - Fonctions superflues - <https://www.owasp.org/index.php/Mobile_Top_10_2016-M10-Extraneous_Functionality>
+- OWASP Mobile Top 10: M10 (Fonctions superflues) - <https://www.owasp.org/index.php/Mobile_Top_10_2016-M10-Extraneous_Functionality>
```
Comment (romualdszkudlarek) [inline Document-fr/0x06-V1-Architecture_design_and_threat_modelling_requireme.md]: - Typo: in French, semi-colons are separated from text with a space (‘ ‘) before and after (while in English there is no space before)

File: Document-fr/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md [hunk 2]
```diff
@@ -48,6 +48,12 @@ Il convient de prendre en compte les considérations suivantes :
...
 | **8.12** | MSTG‑RESILIENCE‑12 | Si le but de l'obscurcissement est de protéger des traitements sensibles, le schéma d'obscurcissement utilisé est à la fois approprié à la tâche considérée et est résistant envers les méthodes de dé-obscurcissement manuelles et automatiques, en prenant en considération les recherches disponibles. L'efficacité du schéma d'obscurcissement doit être validée à travers du test manuel. Il convient de noter que les fonctionnalités d'isolation au niveau matériel doivent être mises en pratique de préférence à l'obscurcissement toutes les fois que cela est possible. | ✓ |
 
+### Entraver l'écoute
+
+| # | MSTG-ID | Description | R |
+| --- | --- | --- | -- |
+| **8.13** | MSTG‑RESILIENCE‑13 | Pour une défense profonde, en plus d'avoir un durcissement solide des parties communicantes, le chiffrement des données utiles au niveau de l'application peut être appliquer pour entraver l'écoute. | ✓ |
```
Comment (romualdszkudlarek) [inline Document-fr/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md]: - 'défense profonde' -> 'défense en profondeur' (I guess you mean 'defense in-depth')
- 'durcissement solide': this is a 'pléonasme' (hard hardening). I'd put it as 'durcissement efficace'
- 'utiles': I guess you mean 'data used by the application'. In that case, I'd put it as 'utilisées'
- 'appliquer' -> 'appliqué'
- in a general manner: I guess you mean that, in addition to using network-level protections (SSL, ...), data encryption may be used at the applicative level to offer an additional layer of protection. In that case, as written today, the paragraph does not reflect this. I'd change it to 'Pour une défense en profondeur, en plus d'établir une liaison sécurisée au niveau réseau, le chiffrement des données au niveau applicatif peut être appliqué pour entraver l'écoute'.

File: Document-fr/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,24 @@
 
 Ce document est généré automatiquement à {{gitbook.time}}
 
+## V1.2-RC 
+
+Les changement suivants font partie de la livraison 1.2:
+
+- Mise à jour des traductions de MASVS dans les langues suivantes: Chinoise (ZHTW), Anglais, Allemande, Français, Russe, Espagnole et Japonaise.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: - 'traductions de MASVS' -> 'traductions du MASVS'
- in French, opposite to English, names of languages are not capitalized: 'les langues suivantes : chinois (ZHTW), anglais, allemand, français, russe, espagnol et japonais.' (or have all names with the feminine form, both may work: chinoise, anglaise, ...). Only names of people get capitalized: 'les Chinois', 'les Anglais', 'les Français', ...

File: Document-fr/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,24 @@
...
+- Promu au statut flagship.
+- Exigence changé: MSTG-STORAGE-1 "doivent être utilisées".
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: - 'changé' -> 'changée'

File: Document-fr/CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,24 @@
...
+- Exigences MSTG-STORAGE-13, MSTG-STORAGE-14, MSTG-STORAGE-15 sont ajoutées avec une concentration sur la protection des données.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: - 'Exigences' -> 'Les exigences'
- 'avec une concentration' -> 'avec un accent'

File: Document-fr/CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,24 @@
...
+- Exigence MSTG-AUTH-11 mise à jour pour garder les informations contextuelles.
+- Exigence MSTG-CODE-4 mise à jour pour couvrir plus que juste le déboggage.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: - 'plus que juste' does not make sense in French -> 'plus que'. I'd better phrase it as 'mise à jour pour avoir une couverture plus large que seulement le déboggage'

File: Document-fr/CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,24 @@
...
+- Exigence MSTG-PLATFORM-10 ajoutée pour protéger de plus l'utilisation des WebViews.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: - 'pour protéger de plus' does not make sense -> maybe 'pour offrir une protection plus large à l'utilisation des WebViews.'?

File: Document-fr/CHANGELOG.md [hunk 8]
```diff
@@ -2,6 +2,24 @@
...
+- Exigence MSTG-AUTH-12 ajoutée pour rappeler les développeurs d'implémenter l'autorisation, surtout pour les applications multi-utilisateurs.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: - 'pour rappeler les' -> 'pour rappeler aux'
- in a general manner: authorization is necessary only in the context of several users using the same application, either remotely (then authorization is mostly needed on the server side) or locally (then authorization is needed locally). So a clear path should be presented to developers. And, just my opinion, I would not 'remind' the use of authorization, but rather make it mandatory, or give a clear choice with criteria for an informed choice of implementing it or not.

File: Document-fr/CHANGELOG.md [hunk 9]
```diff
@@ -2,6 +2,24 @@
...
+- Ajout d'un petit peut sur comment MASVS doit être utiliser à la base d'une evaluation du risk.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: - 'Ajout d'un petit peut sur comment MASVS doit être utiliser à la base d'une evaluation du risk' -> 'Ajout de quelques éléments concernant l'utilisation du MASVS pour une évaluation de risque'

File: Document-fr/CHANGELOG.md [hunk 10]
```diff
@@ -2,6 +2,24 @@
...
+- Ajout d'un petit peut de description sur le contenu payant.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: This is not related to security as is, so maybe better reflect the security aspect of additions, like 'Ajout de quelques éléments concernant la sécurisation du contenu payant'.

File: Document-fr/CHANGELOG.md [hunk 11]
```diff
@@ -2,6 +2,24 @@
...
+- Exigence MSTG-ARCH-11 ajoutée pour inclure une politique de divulgation responsable pour les applications L2.
+- Exigence MSTG-ARCH-12 ajoutée pour montrer aux développeurs qu'ils diovent respecter les lois de confidentialité internationaux.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: - 'diovent' -> 'doivent'
I would not say that developers need to be 'shown' something, but rather need to be 'told', or even 'explained'. Also, instead of referring to international laws on confidentiality, I'd better refer to ' standards and regulations throughout the world that place requirements on confidentiality'. So I'd put it as 'ajoutée pour rappeler les exigences provenant des lois et standards à travers le monde'

File: Document-fr/CHANGELOG.md [hunk 12]
```diff
@@ -2,6 +2,24 @@
...
+- Exigence MSTG-PLATFORM-11 ajoutée pour prévenir l'espionage par des claviers tierces.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: - 'prévenir l'espionage par des claviers tierces' -> 'contrer les menaces pouvant provenir des claviers tiers'

File: Document-fr/CHANGELOG.md [hunk 13]
```diff
@@ -2,6 +2,24 @@
...
+- Exigence MSTG-RESILIENCE-13 pour entraver l'écoute dans l'application.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: As I understand it from the paragraph above, the requirement is meant to offer defense in depth and propose an additional layer of control with encryption at the application level. So that may be like 'pour offrir un niveau de contrôle supplémentaire avec du chiffrement au niveau applicatif afin d'entraver l'écoute par un attaquant.'

File: Document-fr/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md (hunk 2 above)
Comment (A-AFTAHI) [reply Document-fr/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md]: in the original text "next to having solid hardening of the communicating parties", we talk about the "communicating parties" hareding in general and not just the network-level protection. so i kept the previous sentence and i applied the grammar corrections you suggested.

File: Document-fr/0x07-V2-Data_Storage_and_Privacy_requirements.md [hunk 14]
```diff
@@ -35,6 +35,9 @@ La grande majorité des problèmes de divulgation de données peuvent être emp
 | **2.10** | MSTG‑STORAGE‑10 | L'application ne garde pas les données sensibles en mémoire plus longtemps que nécessaire et la mémoire est explicitement nettoyée après son utilisation. |  | ✓ |
 | **2.11** | MSTG‑STORAGE‑11 | L'application met en oeuvre un minimum de politique concernant la sécurité de l'accès à l'appareil tel que l'obligation pour l'utilisateur de définir un code d'accès à l'appareil. |  | ✓ |
 | **2.12** | MSTG‑STORAGE‑12 | L'application instruit l'utilisateur sur les types d'information personnellement identifiable traités ainsi que sur les bonnes pratiques que l'utilisateur devrait suivre en utilisant l'application. |  | ✓ |
+| **2.13** | MSTG‑STORAGE‑13 | Aucune donnée sensible ne doit être stockée localement sur l'appareil mobile. Par contre les données doivent être extraites à partir d'un point terminal distant et stockées seulment sur la mémoire. |  | ✓ |
+| **2.14** | MSTG‑STORAGE‑14 | Si le stockage des données sensibles localement est encore exigé, ces dernières doivent être chiffrées par une clé dérivée d'un stockage matériel qui exige l'authentification. |  | ✓ |
```
[the remark this answers is not in the document]
Comment (A-AFTAHI) [reply Document-fr/0x07-V2-Data_Storage_and_Privacy_requirements.md]: I believe that there is a difference between derived (dérivée) key and retrieved (récupérée) key.
in the original text we have "it should be encrypted using a key derived from hardware backed storage". Which i understand as we're talking about key derivation functions (KDF) if i'm nont mistaken.

File: Document-fr/CHANGELOG.md [hunk 15]
```diff
@@ -6,21 +6,21 @@ Ce document est généré automatiquement à {{gitbook.time}}
 
 Les changement suivants font partie de la livraison 1.2:
 
-- Mise à jour des traductions de MASVS dans les langues suivantes: Chinoise (ZHTW), Anglais, Allemande, Français, Russe, Espagnole et Japonaise.
+- Mise à jour des traductions du MASVS dans les langues suivantes: chinoise (ZHTW), anglais, allemande, français, russe, espagnole et japonaise.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: If the feminine form is chosen for languages, the list becomes 'chinoise (ZHTW), anglaise, allemande, française, russe, espagnole et japonaise'

File: Document-fr/CHANGELOG.md [hunk 16]
```diff
@@ -6,21 +6,21 @@ Ce document est généré automatiquement à {{gitbook.time}}
...
-- Exigences MSTG-STORAGE-13, MSTG-STORAGE-14, MSTG-STORAGE-15 sont ajoutées avec une concentration sur la protection des données.
+- Exigence changée : MSTG-STORAGE-1 "doivent être utilisées".
+- Les exigences MSTG-STORAGE-13, MSTG-STORAGE-14, MSTG-STORAGE-15 sont ajoutées avec un accent sur la protection des données.
 - Exigence MSTG-AUTH-11 mise à jour pour garder les informations contextuelles.
-- Exigence MSTG-CODE-4 mise à jour pour couvrir plus que juste le déboggage.
-- Exigence MSTG-PLATFORM-10 ajoutée pour protéger de plus l'utilisation des WebViews.
-- Exigence MSTG-AUTH-12 ajoutée pour rappeler les développeurs d'implémenter l'autorisation, surtout pour les applications multi-utilisateurs.
-- Ajout d'un petit peut sur comment MASVS doit être utiliser à la base d'une evaluation du risk.
-- Ajout d'un petit peut de description sur le contenu payant.
+- Exigence MSTG-CODE-4 mise à jour pour avoir une couverture plus large que seulement déboggage.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: - 'que seulement déboggage' -> 'que le déboggage seulement' or 'que seulement le déboggage'

File: Document-fr/CHANGELOG.md [hunk 17]
```diff
@@ -6,21 +6,21 @@ Ce document est généré automatiquement à {{gitbook.time}}
...
+- Ajout de quelques éléments concernant la sécurisation du contenu payant.
 - Exigence MSTG-ARCH-11 ajoutée pour inclure une politique de divulgation responsable pour les applications L2.
-- Exigence MSTG-ARCH-12 ajoutée pour montrer aux développeurs qu'ils diovent respecter les lois de confidentialité internationaux.
-- Création d'un style consistant pour toutes les références.
-- Exigence MSTG-PLATFORM-11 ajoutée pour prévenir l'espionage par des claviers tierces.
-- Exigence MSTG-RESILIENCE-13 pour entraver l'écoute dans l'application.
+- Exigence MSTG-ARCH-12 ajoutée pour rappeler les exigences provenant des lois et réglementations à travers le monde.
+- Création d'un style cohérent pour toutes les références.
+- Exigence MSTG-PLATFORM-11 ajoutée pour contrer les menaces pouvant provenir des claviers tiers.
+- Exigence MSTG-RESILIENCE-13 pour pour offrir un niveau de contrôle supplémentaire avec du chiffrement au niveau applicatif afin d'entraver l'écoute par un attaquant.
```
Comment (romualdszkudlarek) [inline Document-fr/CHANGELOG.md]: - 'pour pour' -> 'pour'

File: Document-fr/0x07-V2-Data_Storage_and_Privacy_requirements.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (romualdszkudlarek) [reply Document-fr/0x07-V2-Data_Storage_and_Privacy_requirements.md]: That makes sense. However, the idea may be to retrieve a secret (KEK for instance) from a hardware component (secure enclave in Apple devices for instance, that requires authentication before delivering the key) and derive a key from it, no? Otherwise, some hints on how to derive something from a hardware component may be welcome.

[the remark this answers is not in the document]
Comment (A-AFTAHI) [reply Document-fr/0x07-V2-Data_Storage_and_Privacy_requirements.md]: @commjoen can u please enlighten us about the aim of the MSTG‑STORAGE‑14. 
"a key derived from hardware backed storage which requires authentication." does it mean that the (Key Encryption Key, KEK) is stored in a secure hardware component which requires authentication. and then the other keys will be derived from the 'KEK' or the key is literally derived from the hardware?

[the remark this answers is not in the document]
Comment (romualdszkudlarek) [reply Document-fr/0x07-V2-Data_Storage_and_Privacy_requirements.md]: Based on the original text ("If sensitive data is still required to be stored locally, it should be encrypted using a key derived from hardware backed storage which requires authentication".), the translation seems to make sense...

File: Document-fr/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md (hunk 2 above)
Comment (romualdszkudlarek) [reply Document-fr/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md]: Based on the original text ("As a defense in depth, next to having solid hardening of the communicating parties, application level payload encryption can be applied to further impede eavesdropping."), I'd suggest "Pour une défense en profondeur, en plus d'un durcissement efficace des parties communicantes, le chiffrement des données au niveau applicatif peut être appliqué pour entraver l'écoute.")

File: Document-fr/0x03-Using_the_MASVS.md [hunk 18]
```diff
@@ -14,7 +14,7 @@ Atteindre les exigences du niveau MASVS-L1 amène à une application qui suit le
...
 
-**II: Il faut noter que les contrôles de protection des logiciels présentés dans le MASVS-R et décrits dans l'OWASP MSTG peuvent finalement être contournés et ne doivent jamais être utilisés comme alternative auxs contrôles de sécurité. Par contre, ils sont conçus pour ajouter des contrôles supplémentaires envers des menaces spécifiques pour les applications qui satisfont aux exigences de MASVS-L1 et MASVS-L2.**
+**II: Il faut noter que les contrôles de protection des logiciels présentés dans le MASVS-R et décrits dans l'OWASP MSTG peuvent finalement être contournés et ne doivent jamais être utilisés comme alternative auxs contrôles de sécurité. Par contre, ils sont conçus pour ajouter des contrôles supplémentaires envers des menaces spécifiques pour les applications qui satisfont auxs exigences de MASVS-L1 et MASVS-L2.**
```
Comment (romualdszkudlarek) [inline Document-fr/0x03-Using_the_MASVS.md]: 'auxs' -> 'aux' (2 fois : aux contrôles, aux exigences)

**Comment by commjoen** [comment]
Thank you both!
